raiden/tests/integration/network/transport/test_matrix_transport.py

```diff
@@ -142,7 +142,7 @@ def ping_pong_message_success(transport0, transport1):
     transport1._raiden_service.sign(ping_message)
     transport1.send_async(queueid0, ping_message)
 
-    with Timeout(20, exception=False):
+    with Timeout(40, exception=False):
```

Comment:
Can you make this a constant, so that we have a central location to change these timeouts.

Reply:
@palango I just discussed with @ulope that he'll take care of this in #4333

Reply:
So I would skip the additional CI run if there isn't anything else?

Reply:
Fine for me!